fms/utils/tokenizers.py

```diff
@@ -95,6 +95,8 @@ def convert_ids_to_tokens(self, ids: Union[List[int], torch.LongTensor]):
         return self.sp_model.id_to_piece(ids)
 
     def convert_tokens_to_ids(self, tokens: list[str]):
+        if isinstance(tokens, str):
```

Comment:
Since we're allowing either str or list[str] we should change the type annotation to `Union[str, list[str]]`. This probably needs to be changed in the base class and any other subclasses as well. The base class could probably also use a docstring specifying that a str parameter will be interpreted as a single token.